spec/unit/kiwi_config_spec.rb

```diff
@@ -224,6 +231,21 @@
       expect(config.sh).not_to match(/systemctl \w+-runtime /)
     end
 
+    context "if an empty docker service scope is provided" do
+      it "does not raise" do
+        expect { KiwiConfig.new(system_description_with_docker_services) }.not_to raise_error
+      end
+
+      it "shows a warning" do
+        KiwiConfig.new(system_description_with_docker_services)
+
+        expect(captured_machinery_output).to include(
+          "Warning: Containers do not have an init system, so the package default service" \
```

Comment:
I think it would be sufficient to check for a substring here in order to make the test more robust.